src/components/organisms/UiModal/UiModal.vue

```diff
@@ -464,15 +467,38 @@ const confirmHandler = () => {
 const cancelHandler = () => {
   emit('cancel');
   emit('update:modelValue', false);
+  if (dialog.value) dialog.value.close();
 };
+
 onMounted(() => {
   if (!props.isClosable) return;
+
   window.addEventListener('keydown', keydownHandler);
 });
+
 onBeforeUnmount(() => {
   if (!props.isClosable) return;
   window.removeEventListener('keydown', keydownHandler);
 });
+
```

Comment:
So either 
```suggestion

onMounted(() => {
  if (!props.isClosable) return;
  window.addEventListener('keydown', keydownHandler);
});

onBeforeUnmount(() => {
  if (!props.isClosable) return;
  window.removeEventListener('keydown', keydownHandler);
});

```

Or 
```suggestion

onMounted(() => {
  if (!props.isClosable) return;

  window.addEventListener('keydown', keydownHandler);
});

onBeforeUnmount(() => {
  if (!props.isClosable) return;
  
  window.removeEventListener('keydown', keydownHandler);
});

```
you must decide :D